modules/era/lua_dynamis/mob_spawning_files/dynamis_xarcabard_mobs.lua

```diff
@@ -526,15 +526,7 @@ xi.dynamis.mobList[zoneID][1].wave =
     138, -- (138-D)  Manifest Icon
     139, -- (139-D)  Manifest Icon
     140, -- (140-D)  Avatar Icon
-    141, -- (141-D)  Avatar Icon
-    142, -- (142-Q)  Avatar Icon
-    143, -- (143-G)  Avatar Icon
-    144, -- (144-D)  Avatar Icon
-    145, -- (145-D)  Avatar Icon
-    146, -- (146-D)  Avatar Icon
-    147, -- (147-D)  Avatar Icon
-    148, -- (148-D)  Avatar Icon
-    149  -- (149-D)  Avatar Icon
+    141 -- (141-D)  Avatar Icon
```

Comment:
the lack of extra space makes my heart hurt

Reply:
> the lack of extra space makes my heart hurt

fixed!